src/DIRAC/Core/Utilities/DIRACSingleton.py

```diff
@@ -36,11 +24,9 @@ class DIRACSingleton(type):
     If you want make your class a singleton, just set its  __metaclass__ to
```

Comment:
```suggestion
    If you want make your class a singleton, just set the keyword argument ``metaclass=DIRACSingleton``  in its definition
```
next line also needs update.